tests/management/group/test_view.py

```diff
@@ -1831,8 +1921,10 @@ def test_system_flag_update_on_add(self, send_kafka_message):
             ]
             kafka_mock.assert_has_calls(notification_messages, any_order=True)
 
+    @override_settings(V2_BOOTSTRAP_TENANT=True)
+    @patch("management.relation_replicator.outbox_replicator.OutboxReplicator._save_replication_event")
     @patch("core.kafka.RBACProducer.send_kafka_message")
-    def test_system_flag_update_on_remove(self, send_kafka_message):
+    def test_system_flag_update_on_remove(self, send_kafka_message, mock_method):
```

Comment:
I like that you adjusted this test case to account for still one remaining role in the group. Maybe we should add back a test case for when we remove all the roles in the default group?